tests/foreman/ui/test_repository.py

```diff
@@ -34,7 +35,9 @@
 from robottelo.constants import DataFile
 from robottelo.constants import DOWNLOAD_POLICIES
 from robottelo.constants import INVALID_URL
+from robottelo.constants import PRDS
```

Comment:
Optional, You already imported the constants, so don't explicitly import them, try `constants.PRDS`

Reply:
@Gauravtalreja1 I'm following the convention of what's already in the file.  This shouldn't be a blocker, right?

Reply:
yes, I already mentioned it as optional/non-blocking, you've imported constants here and there is a explicit imports as well